### PR TITLE
[OBMIN-297] Created Toast

### PR DESCRIPTION
**Pull request by Kyslytsia**
Created Toast

### CODE REVIEW

**Review by ASGladchenko — APPROVED** [review]

**Review by AkintsevaElizaveta — APPROVED** [review]

**Review by rpkyrych — CHANGES_REQUESTED** [review]
You should squash your commits and push force to the branch!!!

**Review by rpkyrych — APPROVED** [review]
Rewritten commits with a single one

**Review by Mish-hash — APPROVED** [review]

File: src/main/resources/react/src/components/common/index.js [hunk 1]
```diff
@@ -10,4 +10,3 @@ export { ModalContext } from './pop-up/modal-context';
 export { ModalProvider } from './pop-up/modal-context-provider';
 export { SearchContext, SearchProvider } from './search-context';
 export { showPassword } from './show-password';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/common/index.js:12]: I think you should delete showPassword from EVO

File: src/main/resources/react/src/pages/AddGoods/location/index.jsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
 /* eslint-disable indent */
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/location/index.jsx:1]: try to delete this line

**Review by Wolshebnik — CHANGES_REQUESTED** [review]

File: src/main/resources/react/src/pages/AddGoods/location/index.jsx (hunk 2 above)
Comment (Kyslytsia) [reply src/main/resources/react/src/pages/AddGoods/location/index.jsx:1]: if i remove this line i get errors

Comment (Wolshebnik) [reply src/main/resources/react/src/pages/AddGoods/location/index.jsx:1]: sorry, I understood

**Review by Wolshebnik — APPROVED** [review]

**Pull request merged**